apps/condo/domains/miniapp/schema/AllMiniAppsService.js

```diff
@@ -2,12 +2,30 @@
  * Generated by `createservice miniapp.AllOrganizationAppsService --type queries`
  */
 
+const dayjs = require('dayjs')
 const { GQLCustomSchema } = require('@open-condo/keystone/schema')
 const access = require('@condo/domains/miniapp/access/AllMiniAppsService')
 const { ACQUIRING_APP_TYPE, BILLING_APP_TYPE, APP_TYPES, B2B_APP_TYPE } = require('@condo/domains/miniapp/constants')
 const { find } = require('@open-condo/keystone/schema')
 const { APPS_FILE_ADAPTER } = require('@condo/domains/miniapp/schema/fields/integration')
 
+const priorityCompare = (lhs, rhs) => {
```

Comment:
`lhs` and `rhs` is a little unclear name

Reply:
Its common naming for operators and comparators, especially in backend-world :)
 `lhs` - left hand side, 
`rhs` - right hand side. 

So you can easily reinterpret args from `lhs < rhs`  to `operator<(lhs, rhs)` and so on